matchmaking.py

```diff
@@ -100,25 +109,31 @@ def create_challenge(self, username: str, base_time: int, increment: int, days:
             return ""
 
     def update_daily_challenge_record(self) -> None:
-        # As the number of challenges in a day increase, the minimum wait time between challenges increases.
-        # 0   -  49 challenges --> 1 minute
-        # 50  -  99 challenges --> 2 minutes
-        # 100 - 149 challenges --> 3 minutes
-        # etc.
+        """
+        As the number of challenges in a day increase, the minimum wait time between challenges increases.

```

Comment:
Add as first sentence: "Record timestamp of latest challenge and update minimum wait time."